spec/integrations/groups_spec.rb

```diff
@@ -0,0 +1,76 @@
+require 'rails_helper'
+
+RSpec.describe 'Groups', type: :feature do
+  describe 'Index' do
```

Comment:
- [ ] Excellent addition of unit tests for most of the most important components of the RoR application. 👍🏻 

![image](https://user-images.githubusercontent.com/79058364/221361585-894123ad-7e2c-4bb3-939a-7106066cc8c6.png)

- [ ] Now, let's make sure to also add integration tests with capybara for all the most important components of the RoR application. (movements) ✍🏻